### PR TITLE
backend: delete images on delete test

### PR DESCRIPTION
**Pull request by wais-hundekar**
## Notion ticket link

[Delete Images on Delete Test](https://www.notion.so/uwblueprintexecs/Delete-images-on-delete-test-3a33d884a3ef403899a6957b823d539b)

## Implementation description
* Added `deleteImages` function to `deleteTest`

## Steps to test
1. Ensure images are properly deleted

## What should reviewers focus on?
* 

## Checklist
- [ ] My PR name is descriptive and in imperative tense
- [ ] My commit messages are descriptive and in imperative tense. My commits are atomic and trivial commits are squashed or fixup'd into non-trivial commits
- [ ] I have run the appropriate linter(s)
- [ ] I have requested a review from the PL, as well as other devs who have background knowledge on this PR or who will be building on top of this PR

### CODE REVIEW

File: backend/services/implementations/testService.ts [hunk 1]
```diff
@@ -81,6 +81,15 @@ class TestService implements ITestService {
         throw new Error(`Test ID ${id} not found`);
       }
       if (testToDelete.status === AssessmentStatus.DRAFT) {
+        try {
```
Comment (carissa-tang) [inline backend/services/implementations/testService.ts:85]: hmm can we reorder the deletion (so delete test first, then images)? not a big deal but i think having images hanging in GCP is better than having tests with invalid images

Comment (carissa-tang) [reply backend/services/implementations/testService.ts:85]: oops sorry forgot to press comment lmao

Comment (wais-hundekar) [reply backend/services/implementations/testService.ts:85]: Done!

**Review by carissa-tang — APPROVED** [review]
lgtm!

**Pull request merged**